independent-projects/bootstrap/pom.xml

```diff
@@ -33,11 +33,11 @@
         <jboss-logging.version>3.4.2.Final</jboss-logging.version>
         <jsoup.version>1.14.2</jsoup.version> <!-- wagon-http dependency -->
         <junit.jupiter.version>5.8.1</junit.jupiter.version>
-        <maven-core.version>3.8.1</maven-core.version><!-- Keep in sync with sisu.version -->
-        <sisu.version>0.3.4</sisu.version><!-- Keep in sync with maven-core.version -->
+        <maven-core.version>3.8.4</maven-core.version><!-- Keep in sync with sisu.version -->
+        <sisu.version>0.3.5</sisu.version><!-- Keep in sync with maven-core.version -->
         <maven-plugin-annotations.version>3.6.0</maven-plugin-annotations.version>
-        <maven-resolver.version>1.6.2</maven-resolver.version>
-        <maven-shared-utils.version>3.3.3</maven-shared-utils.version> <!-- changed from the original version due to https://snyk.io/vuln/maven:org.apache.maven.shared:maven-shared-utils@3.2.1 -->
+        <maven-resolver.version>1.6.3</maven-resolver.version>
+        <maven-shared-utils.version>3.3.4</maven-shared-utils.version> <!-- changed from the original version due to https://snyk.io/vuln/maven:org.apache.maven.shared:maven-shared-utils@3.2.1 -->
         <maven-wagon.version>3.4.3</maven-wagon.version>
         <httpcore.version>4.4.14</httpcore.version><!-- Keep in sync with maven-wagon.version (wagon-http 3.4.3 brings in 4.4.13 and 4.4.14) -->
         <plexus-cypher.version>1.7</plexus-cypher.version>
```

Comment:
BTW, this property doesn't seem to be used, afaics.

Reply:
I will check it and remove it if needed, thanks!